windows/installer/MozillaVPN.wxs

```diff
@@ -167,8 +179,17 @@
     <CustomAction Id="LaunchVPNFirstExecution" Impersonate="yes" FileKey="MozillaVPNExecutable" ExeCommand="" Return="asyncNoWait" />
     <CustomAction Id="LaunchVPNAfterUpdate" Impersonate="yes" FileKey="MozillaVPNExecutable" ExeCommand="ui --updated" Return="asyncNoWait" />
     <InstallExecuteSequence>
-       <Custom Action="LaunchVPNFirstExecution" After="InstallFinalize">NOT WIX_UPGRADE_DETECTED</Custom>
-       <Custom Action="LaunchVPNAfterUpdate" After="InstallFinalize">WIX_UPGRADE_DETECTED</Custom>
+      <!-- This will schedule a reboot only if the OS is Windows 10 -->
+      <ScheduleReboot After="InstallFinalize">
```

Comment:
As I understand this, it will schedule a reboot after both fresh installs and upgrades even though we only need it on upgrades. It's not really the worst outcome, just inconvenient.